helm/values.yaml

```diff
@@ -21,47 +21,48 @@ service:
   port: 443
 
 # configure access to the notary server
-notary:
-  # domain to the notary server. can be `null` or non-existant to use
-  # the public Docker Hub notary server
+notaries:
+# public notary form dockerhub
+- name: dockerhub
   host: notary.docker.io
-  # only for insecure notary instances with selfsigned certificates.
-  selfsigned: false
-  # selfsignedCert: |
-  #   -----BEGIN CERTIFICATE-----
-  #   -----END CERTIFICATE-----
+  root_keys:
+  - name: library
+    key: |
+      -----BEGIN PUBLIC KEY-----
+      MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEOXYta5TgdCwXTCnLU09W5T4M4r9f
+      QQrqJuADP6U7g5r9ICgPSmZuRHP/1AYUfOQW3baveKsT969EfELKj1lfCA==
+      -----END PUBLIC KEY-----
```

Comment:
I like, but please add a comment what this key is

Reply:
you should actually be able to do the same for connaisseur and the `k8s.gcr.io` images by providing the keys here and setting `verify: true` below, right? That would be nice, as it might allow us to verify all signatures even our own :-)